docs/src/examples/jump_umap_analysis_with_cosmicqc.py

```diff
@@ -0,0 +1,421 @@
+        df_labeled_outliers,
+        how="inner",
+        left_on=[
+            "Metadata_ImageNumber",
+            "Metadata_ObjectNumber",
+            "Metadata_Plate",
+            "Metadata_Well",
+        ],
+        right_on=[
+            "Metadata_ImageNumber",
+            "Metadata_ObjectNumber",
+            "Metadata_Plate",
+            "Metadata_Well",
+        ],
+    )
+
+    df_feature_selected_with_cqc_outlier_data.to_parquet(parquet_sampled_with_outliers)
+
+else:
+    df_feature_selected_with_cqc_outlier_data = pd.read_parquet(
+        parquet_sampled_with_outliers
+    )
+
+df_feature_selected_with_cqc_outlier_data
+# -
+
+# show our data value counts regarding outliers vs inliers
+df_feature_selected_with_cqc_outlier_data[
+    "analysis.included_at_least_one_outlier"
+].value_counts()
+
+# prepare data for normalization and feature selection
+# by removing cosmicqc and analaysis focused columns.
+df_for_normalize_and_feature_select = df_feature_selected_with_cqc_outlier_data[
+    [
+        col
+        for col in df_feature_selected_with_cqc_outlier_data.columns.tolist()
+        if "cqc." not in col and "analysis." not in col
+    ]
+]
+# show the modified column count
+len(df_for_normalize_and_feature_select.columns)
+
+# +
+parquet_pycytominer_normalized = "./BR00117012_normalized.parquet"
+
+# check if we already have normalized data
+if not pathlib.Path(parquet_pycytominer_normalized).is_file():
+    # normalize the data using pcytominer
+    df_pycytominer_normalized = pycytominer.normalize(
+        profiles=df_for_normalize_and_feature_select,
+        features="infer",
+        image_features=False,
+        meta_features="infer",
+        method="standardize",
+        output_file=parquet_pycytominer_normalized,
+        output_type="parquet",
+    )
+
+# +
+parquet_pycytominer_feature_selected = "./BR00117012_feature_select.parquet"
+
+# check if we already have feature selected data
+if not pathlib.Path(parquet_pycytominer_feature_selected).is_file():
+    # feature select normalized data using pycytominer
+    df_pycytominer_feature_selected = pycytominer.feature_select(
+        profiles=parquet_pycytominer_normalized,
+        operation=[
+            "variance_threshold",
+            "correlation_threshold",
+            "blocklist",
+            "drop_na_columns",
+        ],
+        na_cutoff=0,
+        output_file=parquet_pycytominer_feature_selected,
+        output_type="parquet",
+    )
+
+
+# +
+def generate_umap_embeddings(
+    df_input: pd.DataFrame, cols_metadata: List[str]
+) -> np.ndarray:
+    """
+    Generates UMAP (Uniform Manifold Approximation and Projection)
+    embeddings for a given input dataframe,
+    excluding specified metadata columns.
+
+    Args:
+        df_input (pd.DataFrame]):
+            A dataframe which is expected to contain
+            numeric columns to be used for UMAP fitting.
+        cols_metadata (List[str]):
```

Comment:
```suggestion
        cols_metadata_to_exclude (List[str]):
```

This adds a bit more clarity

Reply:
Thanks, will add this in!